src/common/components/vanilla-modules/forms/form-group-input.js

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
+
+import styles from '../../../style/vanilla/css/forms.css';
+
+export default function FormGroupInput(props) {
+  const { disabled, name, label, type, placeholder } = props;
+  const id = `ID_${name}`;
+  const status = props.touched ? ( props.valid ? 'p-form-validation is-success' : 'p-form-validation is-error') : null;
+
+  return (
+    <div className={ `${styles['p-form__group']} ${styles[status]}` }>
+      <label htmlFor={ id } className={ styles['p-form__label'] }>
+        { label }
+      </label>
+      <div className={ styles['p-form__control'] }>
+        <input
+          id={ id }
+          name={ props.sensitive ? null : name }
+          data-name={ name }
+          type={ type }
+          required={ props.required }
+          disabled={ disabled }
+          placeholder={ placeholder }
+          onChange={ props.onChange }
+          onBlur={ props.onBlur }
+          value={ props.value || '' }
```

Comment:
The same issue is also repeated in other component, so not commenting there on that.